fms/testing/_internal/hf/model_test_suite.py

```diff
@@ -199,6 +201,23 @@ def test_hf_model_round_trip_equivalence(
             HFModelSignatureParams(fms_hf_model_loaded, self._get_hf_signature_params),
         )
 
+    @pytest.mark.skipif(
+        platform.system() != "Linux",
+        reason=f"pytorch compile is stable on Linux, skipping as current platform is {platform.platform()}",
```

Comment:
nit: calling compile stable after the last 6 months is kinda funny to me, but yes, I understand what you mean